pkgs/by-name/gf/gfxstream/package.nix

```diff
@@ -0,0 +1,60 @@
+{ clangStdenv
+, fetchFromGitiles
+, aemu
+, meson
+, pkg-config
+, ninja
+, python3
+, vulkan-headers
+, vulkan-utility-libraries
+, glm
+, libglvnd
+, xorg
+, lib
+, vulkan-loader
+}:
+
+clangStdenv.mkDerivation {
+  pname = "gfxstream";
+  version = "unstable-2023-12-19";
+  src = fetchFromGitiles {
+    url = "https://android.googlesource.com/platform/hardware/google/gfxstream";
+    rev = "5ef37b16f4777f6052903ffcdf96ad0bb87e1572";
+    hash = "sha256-xUKDGTwF05oojuFhs1ruDPmRdOnkGuYo4IK7KdTYZ0k=";
+  };
+  postPatch = ''
+    ln -s common/etc/etc.cpp host/compressedTextureFormats/etc.cpp
+    ln -s common/etc/etc.cpp host/gl/glestranslator/GLcommon/etc.cpp
+  '';
+  preConfigure = ''
+    export NIX_CFLAGS_COMPILE="$NIX_CFLAGS_COMPILE -I$(pwd)/common/etc/include"
+  '';
```

Comment:
Why is all this stuff necessary?  It wasn't for my builds…

Reply:
Looks like they fixed this issue again. It used to not find some header files.